cohort_4/playbook/ACTIVITIES.md

```diff
@@ -880,11 +880,42 @@ shouted out after the demo.
 # Freeform Projects
 
 This assumes the hackers are already in groups and have decided what to work on.
-If they have not yet, refer to [Brainstorming](#brainstorming).
+
+If they have not yet, refer to [Brainstorming](#project-formation).
+
+## Scheduling
+
+- make sure to collect links 10 minutes before demos at around 3:50.
+- make sure you remind people of what time the demos is. It is very easy to forget to mention the time.
+
+## Role Of The Organizer
+
+Walking the rope of how to be effecive during this time period can be hard. On one hand, you want to actively help hackers. At the same time, you don't want to hover and so you stay back and let the hackers do their thing and work indepenently.
+
+You want them to ask questions abut at the same time you don't want them to ask you a question when they never searched for it themself yet.
+
+Being a good organizer is knowing when to step in, and when to hang back.
+If unsure, bias on the side of hanging back.
 
 Every set of freeform time for projects culminates in a demo by the end of the
-day. See the [demo section](#demos) for more details, including what hackers are
-encouraged to have finished.
+day. See the [demo section](#demos) for more details, including what hackers are encouraged to have finished.
+
+### What NOT to do
+
+- drive the conversation
+- just cause you notice sthe hackers doing something in a way that you wouldn't
```

Comment:
Check spelling (if you are using Atom, it has a spell checker plugin).

`cause` & `sthe`